tests/_inductor/utils_inductor.py

```diff
@@ -198,6 +198,22 @@ def compare_with_cpu(fn, *args, atol=0.1, rtol=0.1):
     )
 
 
+# compare with cpu
+def compare_with_pytorch(fn, fn_pytorch, *args, atol=0.1, rtol=0.1):
```

Comment:
@bohnstingl : is this a reasonable error tolerance? It should depend on dtype and typically should be lower.